### PR TITLE
JSON-RPC: Add `eth_createAccessList` RPC call

### PR DESCRIPTION
**Pull request by MariusVanDerWijden**

### CODE REVIEW

**Comment by timbeiko** [comment]
@lightclient @alita-moore can you have a quick look before I merge? TY!

**Review by alita-moore — APPROVED** [review]

File: json-rpc/spec.json [hunk 1]
```diff
@@ -1774,6 +1789,25 @@
...
+          "title": "ALResult",
+          "type": "object",
+          "properties": {
+            "accessList": {
+              "$ref": "#/components/schemas/AccessList"
+            },
+            "gasUsed": {
+              "$ref": "#/components/schemas/Integer"
+            },
+            "vmErr": {
```
Comment (lightclient) [inline json-rpc/spec.json:1805]: Is this field optional? If an error is returned, is `accessList` and `gasUsed` omitted?

**Review by lightclient — COMMENTED** [review]
1 quick comment.

**Comment by lightclient** [comment]
oooooof

**Comment by timbeiko** [comment]
Oops, sorry! Will revert this.

**Comment by timbeiko** [comment]
@MariusVanDerWijden reverted in https://github.com/ethereum/eth1.0-specs/commit/faac97e710ed3b18f9673102fd62b5cbac9a3d8a. 

Can you open a new PR with the same changes and I'll make sure to leave it open until you tell me to merge :-) ? Thanks!

**Pull request merged**